src/test/java/com/znsio/workflow/CreateAndUpdatePost.feature

```diff
@@ -0,0 +1,22 @@
+@prod
+Feature: Create and Update User Posts
+
+  Background:
+    * def userId = generateRandomNumber(3)
+    * def title = generateAlphaNumericRandomString(5)
+    * def body = generateAlphaNumericRandomString(10)
+
+  @createPost
+  Scenario: Create a User post and then Update its title
+    * def createPost = karate.call('classpath:com/znsio/templates/fetchPostAndAlbums.feature@t_createPost', {"userId": userId, "title": title, "body": body, expectedStatus: 201})
+    * karate.log('response : ',createPost.response)
```

Comment:
Where is validation for create post response?

Reply:
validation added